First draft (#1)
temporalio/edu-interacting-with-workflows-python-code: exercises/querying-workflows/README.md

2. Anywhere within the Workflow definition (for example, just before your Signal functions), add a Query function called `current_state_query()`. It should be annotated with `@workflow.query` and `return self._current_state`.
3. Save the file.

## Part B: Performing a Query from a Client

In this part of the exercise, you will create another Temporal client that sends a Query. `queryclient.py` currently contains a near-empty Temporal client. You will use this client to send a Query.

1. Edit the `queryclient.py` file. This time, the `handle` has been obtained for you. 
2. Immediately after the line that obtains the `handle`, use `handle.query()` to send a query. You can import the Query type from `workflow.py` — note that `from workflow import MyWorkflow` is included at the imports at the start of this file. Save the query result in a variable like `result`.

> 3. Immediately after that, add some of logging the Query result. For this exercise, a `print()` statement is fine.

Reviewer: ```suggestion
3. Immediately after that, log the Query result. For this exercise, a `print()` statement is fine.
```
